styles.css

```diff
@@ -0,0 +1,105 @@
+body {
+  background-image: url("./images/background.png");
+  background-size: cover;
```

Comment:
Nice work, using the Figma image as the background. However, I suggest that you use the background image property for the appropriate container and that you fix some of the `background` properties, so as to make the design pixel-perfect. 😀 
You can read about background properties [here](https://www.w3schools.com/cssref/css3_pr_background.php) and [here](https://developer.mozilla.org/en-US/docs/Web/CSS/background)

Also make sure that the padding, margin, font-size, font-weight, font-family, and all other properties matches that of the design on Figma.

- Your design
<img width="167" alt="wrong" src="https://user-images.githubusercontent.com/41163571/219182156-8fabe116-d64a-461d-8e5f-c4ece9585c93.png">

- Figma design
<img width="202" alt="right" src="https://user-images.githubusercontent.com/41163571/219182529-930e10ba-30a9-4f37-88dc-58694d169de2.png">